core/src/com/group66/game/cannon/BallManager.java

```diff
@@ -127,12 +141,14 @@ public void draw(SpriteBatch batch, float runtime) {
 				}
 			}
 			/* Does the ball hit the edge? */
-			if (ball.getX() - ball.getRadius() <= Config.BOUNCE_X_MIN && Math.toDegrees(ball.getAngle()) > 90) {
+			if (ball.getX() - ball.getRadius() <= Config.BOUNCE_X_MIN
```

Comment:
A general question: I noticed that when the line is long, you write it in two lines. Many changes like this were done in this pull request. Is the the effect of some tool you integrated?